packages/asset-swapper/src/network/two_hop_sampler.ts

```diff
@@ -0,0 +1,185 @@
+import { BigNumber } from '@0x/utils';
+
+import { ERC20BridgeSamplerContract } from '../wrappers';
+
+import { Chain } from './chain';
+import { Address, ERC20BridgeSource, DexSample, FillData, MultiHopCallInfo, SourceSampler, SourceSamplerMap } from  './types';
+import { ContractHelper, createContractWrapperAndHelper } from './utils';
+import { ZERO_AMOUNT, MAX_UINT256 } from './constants';
+
+export interface TwoHopFillData extends FillData {
+    firstHop: DexSample;
+    secondHop: DexSample;
+    intermediateToken: Address;
+}
+
+export class TwoHopSampler {
```

Comment:
This sampler class is not like the other samplers. Totally different interface and needs to be called separately in `SassySampler`.